src/controllers/user.ts

```diff
@@ -0,0 +1,28 @@
+import express, { Request, Response } from 'express'
+
+import BaseController from './base'
+import UserModel from '../models/user'
+
+class UserController implements BaseController {
```

Comment:
Why do we need to store the users?

Reply:
- Mapping of visualizations to users will become easier so users will easily be able to see their or someone else's visualization.
- Giving a user model will help in extending the platform to not only use GitHub gist but some other platforms like JSFiddle, etc.